arch/arm/soc/st_stm32/stm32f7/Kconfig.defconfig.stm32f746xg

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST STM32F746XG MCU configuration options
+#
+# Copyright (c) 2018 Yurii Hamann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F746XG
+
+config SOC
+	string
+	default stm32f746xx
```

Comment:
You should use double quotes here: "stm32f476xx".
Seems a recent change from master